src/main/resources/templates/db_scripts/oaipmh/revert_createOaiPmhViewFunction.sql

```diff
@@ -1,19 +1,18 @@
-drop index if exists item_metadata_updateddate_idx;
-
-drop index if exists holdings_record_metadata_updateddate_idx;
-
-drop index if exists audit_instance_createddate_idx;
-
-drop index if exists audit_holdings_record_createddate_idx;
-
-drop index if exists audit_item_createddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.item_metadata_updateddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.holdings_record_metadata_updateddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.audit_instance_createddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.audit_holdings_record_createddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.audit_item_createddate_idx;
+
+drop index if exists ${myuniversity}_${mymodule}.instance_pmh_metadata_updateddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.item_pmh_metadata_updateddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.holdings_record_pmh_metadata_updateddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.audit_instance_pmh_createddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.audit_holdings_record_pmh_createddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.audit_item_pmh_createddate_idx;
```

Comment:
Can you move the `drop index` line to the corresponding `create index` line in createOaiPmhViewFunction.sql?

For example:
```
drop index if exists ${myuniversity}_${mymodule}.item_metadata_updateddate_idx;
drop index if exists ${myuniversity}_${mymodule}.item_pmh_metadata_updateddate_idx;
create index if not exists item_pmh_metadata_updateddate_idx on ...

drop index if exists ${myuniversity}_${mymodule}.holdings_record_metadata_updateddate_idx;
drop index if exists ${myuniversity}_${mymodule}.holdings_record_pmh_metadata_updateddate_idx;
create index if not exists holdings_record_pmh_metadata_updateddate_idx on ...
```
Grouping `drop index` and `create index` increases readability.

Reply:
I agree, but indexes depend of functions, so they can't be deleted before functions are deleted. That's why create and drop are separated.

Reply:
Done.